components/web-components/select/index.tsx

```diff
@@ -0,0 +1,43 @@
+import * as React from 'react'
+
+import * as S from './style'
+
+type ValueType = string | number
+
+interface SelectProps {
+  data: Array<{ value: ValueType, text: string }>
+  currentValue?: ValueType
+  onChange?: (value: ValueType) => unknown
+  ariaLabel?: string
```

Comment:
Since this component doesn't have a label wrapper, and therefore a consumer cannot label it with html, should we make this property required? I think it could be interesting to start that way. The Views framework requires descriptions on everything, and I think this should too.

Reply:
enforcing descriptions for a11y is what i'm always up for